src/app/pages/deposits/no-deposits/no-deposits.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-no-deposits',
```

Comment:
Hey @Genza999, we may not need to create a separate component to display that just one paragraph, we can use an `ng-template structural directive` to show conditional HTML.

Reply:
Thanks @CryceTruly , I have worked on removing this component as it was indeed useless